web/template/attorney_fixtures.gohtml

```diff
@@ -6,6 +6,11 @@
   <p class="govuk-body">Entering an email will cause a reference code to be sent and redirect you to the start page. Leave it blank to be signed in and taken to the dashboard instead.</p>
   {{ template "input" (input . "email" "email" "" "classes" "govuk-input--width-20" "type" "email" "spellcheck" "false" "autocomplete" "email") }}
 
+  {{ template "input" (input . "attorneySub" "Attorney OneLogin sub" .Sub "classes" "govuk-input--width-20") }}
```

Comment:
you're doing `r.FormValue("sub")` in the code for each of these, so these aren't being picked up

Reply:
🤦 good spot